test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js

```diff
@@ -573,4 +573,60 @@ describe('Navigation', () => {
       });
     });
   });
+  describe('GlobalSearch', () => {
+    context('Desktop', () => {
+      it('GlobalSearch Desktop', () => {
+        cy.get('.fd-shellbar__action--desktop .luigi-search__input').should(
```

Comment:
Maybe it's better to rely on custom classes instead of classes from Fundamentals (the same applies to  `globalSearch.js`).